.eslintrc

```diff
@@ -51,8 +51,6 @@
     "wrap-iife": 2,
     // strict mode
     "strict": [2, "never"],
-    // variables
-    "no-unused-vars": 2,
```

Comment:
Is there a reason for removing this rule?

Reply:
don't know where this came from, but it's gone from the new eslint config